app/Models/Game.php

```diff
@@ -157,6 +160,17 @@ public static function boot()
                     }
                 }
             }
+
+            // Double write to taggables for both new and updated games.
+            if ($originalTitle !== $freshGame->title || $game->wasRecentlyCreated) {
```

Comment:
Why isn't the `wasRecentlyCreated` logic required for the `WriteGameSortTitleFromGameTitleAction` call? Shouldn't both dependencies always be updated at the same time?

Reply:
In `createNewGame()`, we have this line:
```php
$game->sort_title = (new ComputeGameSortTitleAction())->execute($title);
```
It's being optimistically written on create rather than written as the product of an event.

Reply:
This does seem like a bit of an inconsistency. It seems like we have three possible paths to choose from:
* Do nothing, keep things as-is.
* Remove that line from `createNewGame()`, handle it in `boot()`.
* Keep that line in `createNewGame()`, and also handle tag syncing in that function too.

After weighing the pros & cons of each, I lean towards moving everything into `boot()`. What do you think?

Reply:
I went with option 2. I've removed the optimistic write in `createNewGame()` so everything is now being handled in `boot()`.